File: services/evaluator/test/health.test.ts
import { describe, expect, it } from "vitest";

import { createHttpApplication } from "../source/inbound/http/application.js";

describe("health endpoint", () => {
  it("returns ok", async () => {
    const application = createHttpApplication();

    const response = await application.request("/health");

    expect(response.status).toBe(200);
    await expect(response.json()).resolves.toEqual({
      status: "ok",
    });
  });
});

File: services/assessor/test/unit/health_test.py
from fastapi.testclient import TestClient

from source.main import run


def test_health_returns_ok() -> None:
    """Verify the health endpoint returns the exact status payload."""
    app = run()
    client = TestClient(app)

    response = client.get("/health")

    assert response.status_code == 200
    assert response.json() == {"status": "ok"}
